_includes/0.4/left_sidebar.html

```diff
@@ -34,7 +34,7 @@
     <div class="left-sidebar-group">
         <div class="left-sidebar-header">Example Applications</div>
         <a href="/docs/0.4/examples/gameroom/">Gameroom</a>
-        <a href="/examples/#hyperledger-grid">Grid on Splinter</a>
+        <a href="/docs/0.4/examples/grid_on_splinter/">Grid on Splinter</a>
```

Comment:
It looks like the actual renaming of the files is not in the correct commit

Reply:
Fixed.